core/include/config.h

```diff
@@ -64,6 +64,7 @@ struct config_t {
     int no_msr_pass_through;
 };
 
-#define HAX_MAX_VCPUS 16
+#define HAX_MAX_VCPUS 128
```

Comment:
@insidiaGithub Do you need more than 16 vCPUs per VM? Have you tested this change? I just wonder if this is a step too far for this PR. Apparently, the underlying logic for limiting the number of vCPUs to 16 is very different from that for limiting the number of VMs, and I need more time to analyze the former and understand the ramifications of this change.